### ISSUE TITLE
Overridable field

### ISSUE THREAD

**Issue by ilgrosso** [ilgrosso/oldSyncopeIdM#252]
_From [mdisabat...@gmail.com](https://code.google.com/u/101550733072735377085/) on December 16, 2011 09:51:01_

What steps will reproduce the problem? 1. Check in the Ldap Connector an Overridable field (for example LDAP Filter for Retrieving Accounts)
2. Edit Ldap Resource

What is the expected output?
Resource Form

What do you see instead?
Last cause: No get method defined for class: class org.syncope.types.ConnConfProperty expression: value

org.apache.wicket.WicketRuntimeException: No get method defined for class: class org.syncope.types.ConnConfProperty expression: value
     at org.apache.wicket.util.lang.PropertyResolver.getGetAndSetter(PropertyResolver.java:499)
     at org.apache.wicket.util.lang.PropertyResolver.getObjectAndGetSetter(PropertyResolver.java:341)
     at org.apache.wicket.util.lang.PropertyResolver.getObjectAndGetSetter(PropertyResolver.java:244)
     at org.apache.wicket.util.lang.PropertyResolver.getValue(PropertyResolver.java:97)
     at org.apache.wicket.model.AbstractPropertyModel.getObject(AbstractPropertyModel.java:134)
     at org.apache.wicket.Component.getDefaultModelObject(Component.java:1640)
     at org.apache.wicket.Component.getDefaultModelObjectAsString(Component.java:1668)
     at org.apache.wicket.markup.html.form.FormComponent.getModelValue(FormComponent.java:1211)
     at org.apache.wicket.markup.html.form.FormComponent.getValue(FormComponent.java:837)
     at org.apache.wicket.markup.html.form.TextField.onComponentTag(TextField.java:108)
     at org.apache.wicket.extensions.ajax.markup.html.autocomplete.AutoCompleteTextField.onComponentTag(AutoCompleteTextField.java:270)
     at org.apache.wicket.Component.internalRenderComponent(Component.java:2488)
     at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1576)
     at org.apache.wicket.Component.internalRender(Component.java:2347)
     at org.apache.wicket.Component.render(Component.java:2275)

_Original issue: http://code.google.com/p/syncope/issues/detail?id=252_

**Comment by ilgrosso** [comment]
_From [chicchiricco](https://code.google.com/u/chicchiricco/) on December 16, 2011 01:09:15_

**Owner:** fabio.ma...@gmail.com  
  **Labels:** Component-UI Milestone-Release-201112 Source-branch-0_7_X

**Comment by ilgrosso** [comment]
_From [fabio.ma...@gmail.com](https://code.google.com/u/109095430973973917901/) on December 19, 2011 01:37:51_

This issue was closed by revision r1331 .

**Status:** Fixed
